src/main/java/pl/javastart/task/Vehicle.java

```diff
@@ -0,0 +1,70 @@
+package pl.javastart.task;
+
+public abstract class Vehicle {
+    protected String brand;
+    protected String model;
+    protected int year;
+    protected double mileage;
+    protected String vinNumber;
```

Comment:
Jeżeli używasz protected, żeby np. w metodzie toString odwoływać się do tych pól bez getterów, to warto by było wydzielić klasy Vehicle, Car, Truck, Motorbike do osobnego pakietu, aby np. z metody main nie dało się do tych pól odwoływać bezpośrednio